handlers.go

```diff
@@ -448,6 +487,10 @@ func (b *BouncerHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	if reqParams.OS == "win" && isWinXpClient {
 		reqParams.Product = sha1Product(reqParams.Product)
 	}
+	// If the user is an "old" stub installer, send a pre-2024-cert-rotation product
+	if isPre2024Stub {
```

Comment:
Unless there is an obvious reason (that I cannot immediately see), we could mimick what was done above:

```suggestion
	if reqParams.OS == "win" && isPre2024Stub {
```

Reply:
What is done above was redirect stub installer downloads to a different product.  Here we want to redirect full installer downloads, so the OS is not always `win` (could also be win64 or win64-aarch64).